lambdas/validatecricheck/src/main/java/uk/gov/di/ipv/core/validatecricheck/validation/CriCheckValidator.java

```diff
@@ -1,28 +1,24 @@
 package uk.gov.di.ipv.core.validatecricheck.validation;
 
-import com.google.gson.Gson;
-import com.nimbusds.jose.shaded.json.JSONArray;
 import com.nimbusds.jose.shaded.json.JSONObject;
 import com.nimbusds.jwt.SignedJWT;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import uk.gov.di.ipv.core.library.domain.ErrorResponse;
 import uk.gov.di.ipv.core.library.exceptions.HttpResponseExceptionWithErrorBody;
 import uk.gov.di.ipv.core.library.persistence.item.UserIssuedCredentialsItem;
-import uk.gov.di.ipv.core.validatecricheck.domain.PassportEvidence;
 
 import java.text.ParseException;
 
 import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.VC_CLAIM;
 
 public class CriCheckValidator {
     private static final Logger LOGGER = LoggerFactory.getLogger(CriCheckValidator.class);
-    private static final Gson gson = new Gson();
     public static final String CRI_ID_UK_PASSPORT = "ukPassport";
+    public static final String CRI_ID_KBV = "kbv";
     public static final String EVIDENCE = "evidence";
     public static final int GPG_45_M1A_STRENGTH_SCORE = 4;
```

Comment:
Don't think you need this static vars for the strenght and validity here anymore as you have them in the new PassportEvidenceValidator class